scripts/helpers/classnames.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Converts an array of classes into a string.
+ *
+ * @param {array} classnamesArray Array of classes.
+ *
+ * @return string
+ */
+export function classnames(classnamesArray) {
+	return classnamesArray.filter(String).join(' ').trim();
```

Comment:
Maybe we could also trim each class.

```suggestion
	return classnamesArray.filter(String).map((class) => class.trim()).join(' ').trim();
```

and also remove empty strings.
```suggestion
	return classnamesArray.filter(String).map((class) => class.trim()).filter((class) => class.length).join(' ').trim();
```

Reply:
`.filter(String)` already removes empty strings in the array. 
Maybe we can first trim each string in the array, then remove empty strings and then join all classes into a single string?